third_party/argo/README.md

```diff
@@ -0,0 +1,61 @@
+# Third Party dependency -- Argo
+
+This folder holds resources for KFP to redistribute <https://argoproj.github.io/projects/argo>
+container images.
+
+There's no code change. KFP only makes sure the images comply with licenses of all its dependencies
```

Comment:
It is a bit unclear to me what we mean with this:
```
There's no code change.
```

Reply:
Understood the confusion, I meant we are not modifying argo workflow code, the redistribution only adds license compliance